### PR TITLE
Add DHC  packages into repo for solving DHC issue

### PR DESCRIPTION
**Pull request by langtaibai**
The commit sloves DHC dependency issue. And will cut in November block.

### CODE REVIEW

File: late/scripts/pool.sh [hunk 1]
```diff
@@ -67,10 +67,15 @@ if [ ! -f /etc/apt/sources.list.d/dell.list ]; then
...
-            cd $dir
-            apt-ftparchive packages ../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
+	    cd $dir
+	    apt-ftparchive packages ../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
         fi
     done
+    #Add DHC packages to solve DHC package denpendency issue
+    if [ -d "/cdrom/dhc/packages" ]; then
+        cd /cdrom/dhc/packages
+	apt-ftparchive packages ../../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
```
Comment (CragW) [inline late/scripts/pool.sh:77]: should be `/cdrom/dhc/packages` instead `$dir`

File: late/scripts/pool.sh [hunk 2]
```diff
@@ -67,10 +67,15 @@ if [ ! -f /etc/apt/sources.list.d/dell.list ]; then
     for dir in /cdrom/debs /isodevice/debs /dell/debs;
     do
         if [ -d "$dir" ]; then
-            cd $dir
-            apt-ftparchive packages ../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
+	    cd $dir
+	    apt-ftparchive packages ../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
```
Comment (CragW) [inline late/scripts/pool.sh:71]: likely indended with tabs versus spaces

**Review by CragW — CHANGES_REQUESTED** [review]

File: late/scripts/pool.sh (hunk 1 above)
Comment (langtaibai) [reply late/scripts/pool.sh]: correct

File: late/scripts/pool.sh (hunk 2 above)
Comment (langtaibai) [reply late/scripts/pool.sh]: Yes，will remove

**Review by CragW — APPROVED** [review]

**Pull request merged**